HomeTask2/Program.cs

```diff
@@ -4,12 +4,67 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
+
 namespace HomeTask2
 {
+
+    
+
     class Program
     {
-        static void Main(string[] args)
+
+                static void Main(string[] args)
```

Comment:
Main method is has wrong indent from the beginning of file. Please use Ctrl +K Ctrl + D combination to auto format current document